.github/workflows/dockerhub-publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Docker Publish
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: "0 3 * * *"
+  workflow_dispatch:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - imageName: projector-webstorm
+            tag: 2020.3.3
+            downloadLink: https://download.jetbrains.com/webstorm/WebStorm-2020.3.3.tar.gz
+          - imageName: projector-idea-c
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/idea/ideaIC-2020.3.4.tar.gz
+          - imageName: projector-idea-u
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/idea/ideaIU-2020.3.4.tar.gz
+          - imageName: projector-clion
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/cpp/CLion-2020.3.4.tar.gz
+          - imageName: projector-goland
+            tag: 2020.3.5
+            downloadLink: https://download.jetbrains.com/go/goland-2020.3.5.tar.gz
+          - imageName: projector-datagrip
+            tag: 2020.3.2
+            downloadLink: https://download.jetbrains.com/datagrip/datagrip-2020.3.2.tar.gz
+          - imageName: projector-phpstorm
+            tag: 2020.3.3
+            downloadLink: https://download.jetbrains.com/webide/PhpStorm-2020.3.3.tar.gz
+          - imageName: projector-pycharm-c
+            tag: 2020.3.5
+            downloadLink: https://download.jetbrains.com/python/pycharm-community-2020.3.5.tar.gz
+          - imageName: projector-pycharm-p
+            tag: 2020.3.5
+            downloadLink: https://download.jetbrains.com/python/pycharm-professional-2020.3.5.tar.gz
+          - imageName: projector-rider
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/rider/JetBrains.Rider-2020.3.4.tar.gz
+          - imageName: projector-rubymine
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/ruby/RubyMine-2020.3.4.tar.gz
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: projector-server
+
+      - name: Checkout docker repo
+        uses: actions/checkout@v2
+        with:
+          repository: JetBrains/projector-docker
+          path: projector-docker
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - run: |
+          cd projector-docker
+          ./build-container.sh ${{ secrets.DOCKER_USERNAME }}/${{ matrix.imageName}}:latest ${{ matrix.downloadLink }}
+          docker push ${{ secrets.DOCKER_USERNAME }}/${{ matrix.ImageName }}:latest
+
+      - run: |
+          cd projector-docker
+          ./build-container.sh ${{ secrets.DOCKER_USERNAME }}/${{ matrix.imageName}}:${{ matrix.tag }} ${{ matrix.downloadLink }}
+          docker push ${{ secrets.DOCKER_USERNAME }}/${{ matrix.ImageName }}:${{ matrix.tag }}
```

Comment:
Is there any reason to build it the second time? I suggest reusing the last build. This should at least save time.

Web searched: https://stackoverflow.com/questions/25211198/docker-how-to-change-repository-name-or-rename-image#25214186

Reply:
Yes, sorry. Forgot about the docker tag function.
Fixed